ccmlib/scylla_node.py

```diff
@@ -1230,8 +1230,8 @@ def no_pending_flushes() -> bool:
             raise NodeError("Node %s still has pending flushes after "
                             "%s seconds" % (self.name, wait_timeout))
 
-    def flush(self, ks=None, table=None):
-        super(ScyllaNode, self).flush(ks, table)
+    def flush(self, ks=None, table=None, **kwargs):
+        super(ScyllaNode, self).flush(ks, table, **kwargs)
```

Comment:
the timeout parameter can be passed down to `_wait_no_pending_flushes` too,
but can be done as a follow up.

Reply:
Let's added when some code actually needs it